docs/content/Configuration.md

```diff
@@ -34,6 +34,12 @@ We recommend just setting the base ZK path and the ZK service host, but all ZK p
 
 See the [Zookeeper](ZooKeeper.html) page for more information on configuration options for ZK integration.
 
+### General Logging
+See [Logging](Logging.html) for more information
+|Property|Description|Default|Recommended|
+|--------|-----------|-------|-----------|
+|`java.util.logging.manager`|The logging manager class (this is a standard java config)| `null <<java default>>`|`org.apache.logging.log4j.jul.LogManager`
```

Comment:
this should not be part of the config docs, since this has to be provided as a system property and cannot be passed as part of the runtime.properties file.